packages/eslint-config-airbnb/README.md

```diff
@@ -12,7 +12,14 @@ We export three ESLint configurations for your usage.
 
 Our default export contains all of our ESLint rules, including ECMAScript 6+ and React. It requires `eslint`, `eslint-plugin-import`, `eslint-plugin-react`, and `eslint-plugin-jsx-a11y`.
 
-1. Ensure packages are installed with correct version numbers by running:
+1. Install the correct versions of each package, which are listed by the command:
+
+  ```sh
+  npm info "eslint-config-airbnb@latest" peerDependencies
+  ```
+
+  Bash users can simply run
```

Comment:
This actually works on all linux shells - probably best to say "Non-windows users"